media-fonts/t1utils/t1utils-1.41.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="Utilities for manupulating Adobe Type 1 font software"
+DESCRIPTION="T1utils is a collection of simple programs for manipulating \
+PostScript Type 1 fonts. Together, they allow you to convert between PFA \
+(ASCII) and PFB (binary) formats, disassemble PFA or PFB files into human-\
```

Comment:
Maybe start new paragraphs on new lines like:
```diff
-PostScript Type 1 fonts. Together, they allow you to convert between PFA \
-(ASCII) and PFB (binary) formats, disassemble PFA or PFB files into human-\
+PostScript Type 1 fonts.
+Together, they allow you to convert between PFA (ASCII) and PFB (binary) \
formats, disassemble PFA or PFB files into human-readable ...
```

Reply:
I have copied the description directly from the [README at github](https://github.com/kohler/t1utils/blob/master/README.md#t1utils) but I reformatted it nonetheless.

Reply:
That's OK, it just looks nicer in HaikuDepot :)